instagram/src/components/CommentSection/Comment.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { UserName } from '../PostContainer/PostHeader';
+import styled, { css } from 'styled-components';
+
+export const FlexWrapper = styled.div`
```

Comment:
Nice styled components. I would prefer to see these abstracted to a different file so that you're not muddying your code in this file with all this styling information. Makes it harder to read for other developers when they're looking inside this file for the first time.